### PR TITLE
provided db schemas in a tool folder

### PR DESCRIPTION
**Pull request by HeyMaximus**

### CODE REVIEW

File: db-schemas/products.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+GET /products
+[{
+    id: Number,
+    name: String,
+    slogan: String,
+    description: String,
+    default_price: String
+}, ...]
+
```
Comment (MiladMoulayi) [inline db-schemas/products.js:9]: Looks good! 👍🏾

File: db-schemas/cart.js
Comment (gocodezhang) [inline db-schemas/cart.js:1]: Test

File: db-schemas/reviews.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      "helpfulness": Number,
+      "photos": [{
+          "id": Number,
+          "url": String
+        },
+        {
+          "id": Number,
+          "url": String
+        }, ...]
+    }, ...]
```
Comment (MiladMoulayi) [inline db-schemas/reviews.js:25]: Good to put the ... , we don't need every single review. 👍🏾

File: db-schemas/cart.js
Comment (gocodezhang) [reply db-schemas/cart.js:1]: looks good

File: db-schemas/cart.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+GET /cart
+[
+  {
```
Comment (gocodezhang) [inline db-schemas/cart.js:3]: looks good

**Review by MiladMoulayi — APPROVED** [review]
Looks clean! Well done.

**Review by gocodezhang — APPROVED** [review]
looks good

File: db-schemas/products.js (hunk 1 above)
Comment (HeyMaximus) [reply db-schemas/products.js:9]: OKay. Will address tomorrow ASAP.

File: db-schemas/cart.js (hunk 3 above)
Comment (HeyMaximus) [reply db-schemas/cart.js:3]: Oh! You're so right. Will edit!

Comment (rul008) [reply db-schemas/cart.js:3]: really good

File: db-schemas/products.js (hunk 1 above)
Comment (rul008) [reply db-schemas/products.js:9]: good

**Pull request merged**